concepts/analysis_intro.md

```diff
@@ -1,6 +1,55 @@
-# Work in Progress
-
 This article is under development.
-The work is being tracked in 
+The work is being tracked in
 [this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1341706).
 
+=======
+= Work in Progress
+
+This article is under development.
+The work is being tracked in
+link:https://bugzilla.mozilla.org/show_bug.cgi?id=1341706[this bug].
+
+= Getting Started with Firefox Data
+// Introduces:
+//   * Pings
+
+Firefox clients out in the wild send us data as *pings*. Most of our pings contain some combination of *environment* data (e.g. operating system, hardware, Firefox version) and *probe* data (e.g. max number of open tabs, time spent running in Javascript garbage collection). We have quite a few different pings, but most of our data for Firefox Desktop comes in from link:http://gecko.readthedocs.io/en/latest/toolkit/components/telemetry/telemetry/data/main-ping.html[Main pings].
+
+== Probes
+// Introduces:
+//   * Histograms
+//   * Scalars
+
+When we need to measure specific things about clients, we use probes. A single ping will send in many different probes. There are two types of probes: *Histograms* and *Scalars*.
```

Comment:
I think of probes as a tool to collect additional data, e.g. the code used to populate a field with a measurement. I would consider *Histograms* and *Scalars* to be data types.